mne/io/pick.py

```diff
@@ -12,10 +12,16 @@
 
 from .constants import FIFF
 from ..utils import (logger, verbose, _validate_type, fill_doc, _ensure_int,
-                     _check_option)
+                     _check_option, deprecated)
 
 
+@deprecated('The function mne.io.pick.get_channel_types() has changed names. '
+            'Please use mne.io.pick.get_channel_type_constants() instead.')
 def get_channel_types():
+    return get_channel_type_constants()
+
+
+def get_channel_type_constants():
```

Comment:
Should we add this to `python_reference.rst` or no?

Reply:
Personally I don't see its utility for the average user... If you disagree feel free to push that addition